repos/fdbt-site/src/pages/products/productDetails.tsx

```diff
@@ -312,6 +312,12 @@ const createProductDetails = async (
             content: [ticket.zoneName],
             editLink: '/csvZoneUpload',
         });
+        productDetailsElements.push({
+            id: 'stops',
+            name: 'Stops',
+            content: [ticket.stops.map((stop) => stop.stopName).join(', ')],
```

Comment:
We don't want to do this - some zones will be hundreds and hundreds of stops. We just need the count.